package.json

```diff
@@ -62,7 +62,7 @@
     "multer": "^1.0.1",
     "node-uuid": "~1.4.3",
     "npm-run-all": "^1.5.1",
-    "nunjucks": "^2.3.0",
+    "nunjucks": "2.4.3",
```

Comment:
mostly just "add `^` back in". We should not need to peg the specific 2.4.0 version.